packages/core/config/babel-preset.js

```diff
@@ -57,7 +63,7 @@ module.exports = () => ({
           'babel-plugin-module-resolver',
           {
             alias: {
-              src: './src',
+              src: path.join(redwoodPaths.api.base, 'src'),
```

Comment:
I think this might cause problems when we build since it'll reference the absolute path.

Reply:
Hmm, I think this is leftover from my previous approach. I'll remove it.

Reply:
If this needs to stay, then we're going to have to make sure that the build command still works:

When you run `yarn build api` the transpiled JS should not include absolute paths.